tests/fixtures/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Klaatu Test Page</title>
+    </head>
+    <body>
+        <h1>This is a test page for Klaatu tests :)</h1>
+        <h2>Search elements</h2>
+        <ul>
+            <li id="copy-paste-string">Mozilla</li>
```

Comment:
not really relevant to anything, but doesn't look like this is used [yet?]

Reply:
Will be changing the tests that use "http://allizom.org" to use this page as well